source/gba.cpp

```diff
@@ -14652,6 +14657,25 @@ int cheatsCheckKeys(u32 keys, u32 extended)
   return ticks;
 }
 
+
+
+void cheatListUpdate() {
+	for(int x = 0; x < cheatsNumber; x++) {
+		if(cheatsList[x].enabled)
+			snprintf(cheatsStringList[x], CHEAT_LIST_ENTRY_SIZE, "%s (%s) Status: Enabled", cheatsList[x].desc, cheatsList[x].codestring);
+		else
+			snprintf(cheatsStringList[x], CHEAT_LIST_ENTRY_SIZE, "%s (%s) Status: Disabled", cheatsList[x].desc, cheatsList[x].codestring);
+	}
+	snprintf(cheatsStringList[cheatsNumber], CHEAT_LIST_ENTRY_SIZE, "Add Cheat");
+}
+
+void cheatListInit() {
+	for(int i = 0; i < CHEAT_LIST_SIZE; i++)
+		cheatsStringList[i] = (char*) malloc(CHEAT_LIST_ENTRY_SIZE);
```

Comment:
the memory is allocated but never freed. How about adding a cheatListDeinit function?

Reply:
Why? The only case where this would happen is right before the application quits.

Reply:
it's very likely that the memory is freed up when the process exits(also it's just a very small amount of memory), but there are no guarantees that this will always be the case

Reply:
Do we even have something that's reliably being called on application-exit?